packages/conformance/src/commands/verify-bundle.ts

```diff
@@ -78,32 +68,6 @@ export default class VerifyBundle extends Command {
   }
 }
 
-// Initialize TrustMaterial from TUF
-async function trustMaterialFromTUF(staging: boolean): Promise<TrustMaterial> {
-  const opts: tuf.TUFOptions = {};
-
-  if (staging) {
-    // Write the initial root.json to a temporary directory
-    const tmpPath = await fs.mkdtemp(path.join(os.tmpdir(), 'sigstore-'));
-    const rootPath = path.join(tmpPath, 'root.json');
-    await fs.writeFile(rootPath, Buffer.from(TUF_STAGING_ROOT, 'base64'));
-
-    opts.mirrorURL = TUF_STAGING_URL;
-    opts.rootPath = rootPath;
-  }
-
-  const trustedRoot = await tuf.getTrustedRoot(opts);
-  return toTrustMaterial(trustedRoot);
-}
-
-// Initialize TrustMaterial from a file
-async function trustMaterialFromPath(path: string): Promise<TrustMaterial> {
-  const trustedRoot = await fs
-    .readFile(path)
-    .then((data) => JSON.parse(data.toString()));
-  return toTrustMaterial(TrustedRoot.fromJSON(trustedRoot));
-}
-
```

Comment:
We need these functions for both `verify` and `verify-bundle` now so they're being extracted into a separate file.